test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SchemaCompare/SchemaCompareServiceTests.cs

```diff
@@ -1594,6 +1629,23 @@ public async Task VerifySchemaCompareWarningsBeingExcluded()
             }
         }
 
+        private async Task VerifyContentAndCleanupAsync(string outputFilePath, string patternToMatch)
```

Comment:
Consider renaming `patternToMatch` to `textToMatch` since it's not doing a regex pattern match